auth-web/src/components/auth/manage-business/StartNewBusinessHelp.vue

```diff
@@ -0,0 +1,167 @@
+              <strong>"Open Name Request"</strong> button next to your Name Request.
+            </li>
+            <h4 class="my-5">
+              Use a Numbered Company
+            </h4>
+            <li>
+              To start a numbered corporation in B.C., click the <strong>"Get Started with a B.C. Based
+                Business"</strong> button and follow the instructions.
+            </li>
+          </ul>
+        </li>
+        <li class="mt-10">
+          <h4>Incorporate or Register Your Business</h4>
+          <ul class="mt-4 ml-2 ml-n6 text-align">
+            <li>
+              For named businesses, once your Name Request has been approved and added to your table on
+              this screen, you must select the <strong>"Register Now"</strong> button beside the name to begin
+              your incorporation or registration.
+            </li>
+            <li>
+              Follow the steps in the application and complete all of the required information including addresses,
+              contact information, people and roles, and share structure (if applicable).
+            </li>
+            <li>
+              Retain a copy of all incorporation or registration documents for your business' records.
+            </li>
+          </ul>
+        </li>
+
+        <li class="mt-10">
+          <h4>Manage and Maintain Your Business</h4>
+          <ul class="mt-4 ml-2 ml-n6 text-align">
+            <li>
+              Once your business is incorporated or registered you are required to keep information about your
+              business up to date with the Business Registry.
+            </li>
+            <li>
+              From your table, click <strong>"Manage Business"</strong> to manage and maintain your
+              business information including:
+            </li>
+            <ul>
+              <li>
+                View and change business information.
+              </li>
+              <li>
+                See when annual reports are due and file them each year (if applicable).
+              </li>
+              <li>
+                See the history of your business' filings and download copies of all documents.
+              </li>
+              <li>
+                Dissolve your business.
+              </li>
+            </ul>
+          </ul>
+        </li>
+      </ol>
+    </div>
+  </section>
+</template>
+
+<style lang="scss" scoped>
+@import '$assets/scss/theme.scss';
+ol.bold li::marker {
+  font-weight: bold;
+  font-size: $px-16 !important;
+  color: $gray9;
+  margin:5px 20px 5px 0px;
+}
+
+ul {
+  list-style-type: none;
+}
+
+ul li {
+  margin-bottom: 10px;
+}
+
+ul li::before {
+  content: "•";
+  font-style:bold;
+  line-height: 1em;
+  margin:5px 20px 10px 0px;
+}
+
+li ul li:before {
+  float: left;
+}
+
+li ul ul li:before {
+  float: none;
+  margin-bottom: 50px;
+}
+
+ul ul li {
+  margin-bottom: 5px;
+}
+
+.open-in-new-icon {
+  font-size: 16px;
```

Comment:
This lines is a third size spec you give this icon. (The other is class "small-icon" and "size=16".)

Could you use one of the [v-icon props](https://v2.vuetifyjs.com/en/api/v-icon/#props) to set its size,m or at least remove the 2 specs that don't work?

Reply:
Using v-icon size. removed class. Missed out on removing this earlier

Reply:
Do you still need class "small-icon" on line 21?

Reply:
You are right. nope do not need it